static/components/ExpectedChanges.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import api from 'app/api';
+
+import TaskSummary from './TaskSummary';
+
+function ExpectedChanges({app, env}) {
+  const [lastDeploy, setLastDeploy] = React.useState(undefined);
+
+  const fetchLastDeploy = React.useCallback(async () => {
+    const deploysResp = await api.request('/deploys/', {
+      query: {app, env, status: 'finished'},
+    });
+
+    if (deploysResp.ok) {
+      const deploys = await deploysResp.json();
+      setLastDeploy(deploys[0] ?? null);
+    }
+  }, [app, env]);
+
+  React.useEffect(() => void fetchLastDeploy(), [fetchLastDeploy]);
+
+  if (lastDeploy === undefined) {
+    return (
+      <div>
+        <label>Previous Deploy</label>
+        <p>Loading previous deploy info</p>
+      </div>
+    );
+  }
+
+  if (lastDeploy === null) {
```

Comment:
I don't love using undefined/null for loading/empty states (not very explicit) but it works 🤷